packages/core/src/components/button/test/icon/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Button - Icon</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
+    <link rel="stylesheet" href="../../../../../dist/tegel/tegel.css" />
+    <script type="module">
+      import { defineCustomElements } from '../../../../../loader/index.es2017.js';
+      defineCustomElements();
+    </script>
+  </head>
+
+  <body>
+    <tds-button id="basicButton" text="Button">
+      <tds-icon slot="icon" size="20px" name="chevron_right"></tds-icon>
+    </tds-button>
+  </body>
```

Comment:
Please double check this with @TatyanaBumazhkina and [test case spreadsheet.](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=rl48tp)

Scenario no.10 I understood as "Only icon button with no text" on it so please, double check it :)

Reply:
Aaah yes ofc :) Thats is what that meant. Im so new for these components and I never use a button with only a component but mostly with combination of texts. I will check the testa for only a icon and look for documentation on how to do that. Thanks for checking.

Reply:
@theJohnnyMe updated, And also found that lg and sm should have fixed height and width for buttons with an icon so I added tests for that. atleast for lg size.